.github/actions/spiced-cuda-build/action.yaml

```diff
@@ -14,11 +14,15 @@ inputs:
   target_os:
```

Comment:
@kczimm  also need to add to the nightly container build: https://github.com/spiceai/spiceai/actions/workflows/spiced_docker_nightly.yml